Rakefile

```diff
@@ -13,3 +13,28 @@ begin
   load 'jasmine/tasks/jasmine.rake'
 rescue LoadError
 end
+
+# Clear noisy and unusable tasks added by rspec-rails
+if defined?(RSpec)
+  task("spec").clear
+  %w(
+    automation
+    controllers
+    helpers
+    initializers
+    lib
+    mailers
+    migrations
+    models
+    presenters
+    replication
+    requests
+    routing
+    services
+    task_helpers
+    tools
+    views
+  ).each do |type|
```

Comment:
Fighting the gem's "normal" tasks seems pretty bad form to me (though we obviously do it *because reasons*), so I doubt there's anything that'll make it particularly elegant.